### DISCUSSION TITLE
How to use remark plugin for markdown contents in Zettlr?

### DISCUSSION THREAD

**Discussion by liblaf** [Zettlr/Zettlr#4977] [Q&A] [answered]
I surmise from `package.json` that Zettlr uses `remark` to render markdown. Is there a way to configure `remark` options and add a custom plugin, e.g., [remarkjs/remark-textr](https://github.com/remarkjs/remark-textr)?

**Comment by nathanlesage** [comment, marked as answer]
Zettlr uses remark exclusively for the linter. Any Markdown->HTML conversion is either performed by Pandoc, or by utilizing the parser state of CodeMirror.
